src/Navigation/navigation.js

```diff
@@ -0,0 +1,11 @@
+import "./navigation.css";
+
+const Navigation = () => {
+  return (
```

Comment:
If the component don't have any code you can remove extra "return".
It will be something like this:
```
const Navigation = () => (
<div> test</div>
)
```

Reply:
кога я убираю Ретурн, оно не отрисовывает ни чего